rs/sdk/src/oc_api/client_factory/command/send_message.rs

```diff
@@ -35,11 +33,6 @@ impl<R: Runtime> SendMessageBuilder<R> {
         self
     }
 
-    pub fn with_ephemeral(mut self, ephemeral: bool) -> Self {
```

Comment:
Maybe we keep this in, but we add a `build_message` or something like that which returns the message without sending it to the OC backend?